docs/spec/governance.rst

```diff
@@ -6,124 +6,109 @@ Governance
+On-chain governance is useful for controlling system parameters, authorizing trusted oracles and upgrading the core protocols. The architecture adopted by interBTC is modelled on Polkadot with some significant changes:
 
-.. figure:: ../figures/spec/governance.jpeg
-    :alt: Governance Architecture
-
-
-Parameters (Council)
-~~~~~~~~~~~~~~~~~~~~
-
-
-MaxProposals
-------------
-
-The maximum number of proposals allowed in the queue.
-
-.. note:: **Default Value**: 100
-
-
-MaxMembers
-----------
-
-The maximum number of participants allowed in the council.
-
-.. note:: **Default Value**: 100
+- **Optimistic Governance**
+    - No **Council**, only public proposals from community
+    - Community can elect a **Technical Committee** to fast-track proposals
+    - Referenda are Super-Majority Against (Negative Turnout Bias) by default
 
+- **Stake-To-Vote**
+    - Adopted from Curve's governance model
+    - Users lock the native governance token
+    - Longer lockups give more voting power
 
-Parameters (Voting)
-~~~~~~~~~~~~~~~~~~~
-
-
-EnactmentPeriod
----------------
-
-The period to wait before any approved change is enforced.
-
-.. note:: **Default Value**: 1 Day
-
-
-LaunchPeriod
-------------
-
-The interval after which to process a new referenda from the queue.
-
-.. note:: **Default Value**: 2 Days
+.. figure:: ../figures/spec/governance.jpeg
+    :alt: Governance Architecture
+    :scale: 30
+    :align: center
 
+An important distinction is the ``negative turnout bias`` (Super-Majority Against​) voting threshold. This is best summarized by the `Polkadot <https://wiki.polkadot.network/docs/learn-governance>`_ docs:
 
-VotingPeriod
-------------
+    A heavy super-majority of nay votes is required to reject at low turnouts, but as turnout increases towards 100%, it becomes a simple majority-carries as below.
 
-The period to allow new votes for a proposal or referenda (**Council**).
+    .. math::
 
-.. note:: **Default Value**: 2 Days
+        \frac{ against }{ \sqrt{ electorate } } < \frac{ approve }{ \sqrt{ turnout } }
 
+Terminology
+~~~~~~~~~~~
 
-FastTrackVotingPeriod
----------------------
+- **Proposals** are community-supported motions to perform system-level actions.
+- **Referenda** are accepted proposals undergoing voting.
 
-The period to allow new votes for a proposal or referenda (**Technical Committee**).
+Processes
+~~~~~~~~~
 
-.. note:: **Default Value**: 3 Hours
+Proposals
+---------
 
+1. Account submits public proposal with deposit (``> MinimumDeposit``)
+2. Account "seconds" proposal with additional deposit
+3. New referenda are started every ``LaunchPeriod``
+4. Community can vote on referenda for the ``VotingPeriod``
+5. Votes are tallied after ``VotingPeriod`` expires
+6. System update executed after ``EnactmentPeriod``
+7. Token voters can unlock balance after ``end + EnactmentPeriod * conviction``
```

Comment:
Locking is independent of proposals. Voters simply lock until block X and this determines the voting power (decreases with every block as we get closer to block X).